integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -225,9 +230,9 @@ def test_multiple_granules_happy_path(self):
                 "Expected API output not returned.",
             )
             # Make sure all given granules are present without checking order.
-            self.assertCountEqual(
-                expected_catalog_output_granules,
-                catalog_output_json["granules"],
+            self.assertEqual(
+                len(expected_catalog_output_granules),
+                len(catalog_output_json["granules"]),
```

Comment:
One would think these two would be identical, but apparently not. `assertCountEqual` was evaluating to `0` even when elements were present.